frontend/src/pages/forms/Form.jsx

```diff
@@ -91,6 +97,19 @@ const Form = ({ walletId, setWalletId }) => {
                     <div className="submit">
                         <button type="submit" className='form-button'>Submit</button>
                     </div>
+                    <div className="card-gradients forms-gradient">
+                        <div className="card-gradient"></div>
+                        <div className="card-gradient"></div>
+                    </div>
```

Comment:
create a separate component called CardGradients, that accepts className prop, which is used alongside with "card-gradients", (seems like similar code is used in other places, the only different is this second className, in this case it is `forms-gradient`)

Reply:
Please check if I understood correctly.

Reply:
yes, thx